lib/Github/Api/PullRequest/Review.php

```diff
@@ -159,11 +159,22 @@ public function submit($username, $repository, $pullRequest, $id, array $params
      * @param string $repository  the repository
      * @param int    $pullRequest the pull request number
      * @param int    $id          the review id
+     * @param string  $message    a mandatory dismissal message
      *
      * @return array|string
      */
-    public function dismiss($username, $repository, $pullRequest, $id)
+    public function dismiss($username, $repository, $pullRequest, $id, $message)
```

Comment:
@acrobat Let's add your review, so we can merge this asap (if you agree w/ this one).

Thanks :)